conan/tools/gnu/autotoolstoolchain.py

```diff
@@ -123,7 +123,12 @@ def defines(self):
 
     def environment(self):
         env = Environment()
-        if is_msvc(self._conanfile):
+        compilers = self._conanfile.conf.get("tools.build:compilers", default={}, check_type=dict)
+        if compilers:
+            for compiler, compiler_path in compilers.items():
+                # CC, CXX
+                env.define(compiler.upper(), compiler_path)
```

Comment:
Do we want to define all the "tools.build:compilers" values to environment variables directly?
for example the dict could be:
``
 {"Fortran": "path/to/fortran"}
``
because the var you want to set for CMake is `CMAKE_Fortran_COMPILER`
but for Autotools would be something like `FC` or `F77` ?